Vagrantfile

```diff
@@ -3,18 +3,20 @@
 
 VAGRANTFILE_API_VERSION = "2"
 
+IP_ADDRESS = "10.10.10.6"
```

Comment:
Is hardcoding this ok or should we use hostnames (or configurable values)?

Reply:
This is the IP address for the vagrant machine. There is no way (that I know of) to configure this via a command and is better to have the IP set to something known. Since the flow is to run `vagrant up` this should be fine.